src/main/java/com/yash/ExpenseClaims/controllers/FileUploadController.java

```diff
@@ -0,0 +1,49 @@
+package com.yash.ExpenseClaims.controllers;
+
+import com.yash.ExpenseClaims.dto.ExpenseDto;
+import com.yash.ExpenseClaims.repositories.UploadFilesToDB;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.ModelAndView;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+@RestController
+public class FileUploadController {
+
+    @Autowired
+    private UploadFilesToDB uploadFilesToDB;
+
+    @PostMapping("/upload")
+    public ModelAndView handleFileUpload(
```

Comment:
:x:  if its a post request you should take all the request info in [request body](https://www.baeldung.com/spring-request-response-body), why do we need request params use